curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-recursion-by-building-a-decimal-to-binary-converter/645ca62092c53f29f4ea9bf6.md

```diff
@@ -16,7 +16,7 @@ First, rename the `countdown()` function to `countDownAndUp()`. Remember to upda
 You should not have a function named `countdown()` in your code.
 
 ```js
-assert.notMatch(code, /(var|let|const)\s+countdown\s*=\s*\(\s*\)\s*=>\s*\{[\s\S]+\}/);
+assert.notMatch(code, /(var|let|const)\s+countdown\s*=\s*\(\s*number\s*\)\s*=>\s*\{[\s\S]+\}/);
```

Comment:
This is fine, but it feels a little too restrictive. Similar to what I mention on the other PR where users had to delete some code. This will pass by just changing `number` to `numer`. What about something like this: `assert.notTypeOf(countdown, 'function');`

Reply:
`assert.isNotFunction(countdown)` might even better show the test intent.